packages/x-archetype-utils/src/home/home-template.html

```diff
@@ -83,24 +83,20 @@
           class="x-archetype-header__input"
```

Comment:
is the lens so big?
![image](https://user-images.githubusercontent.com/6597815/211846668-4a42232a-8f98-4df7-8bf0-1f360f948ba8.png)